library/sha256.c

```diff
@@ -180,7 +201,11 @@ static int mbedtls_a64_crypto_sha256_determine_support(void)
 
     if (setjmp(return_from_sigill) == 0) {         /* First return only */
         /* If this traps, we will return a second time from setjmp() with 1 */
+#if defined(MBEDTLS_ARCH_IS_ARM64)
         asm ("sha256h q0, q0, v0.4s" : : : "v0");
+#else
+        asm ("sha256h.32 q0, q0, q0" : : : "q0");
```

Comment:
Would it make sense to put the `volatile` on these two lines? (I was going to raise a PR to do this, but it would conflict with this one)

Reply:
DId you mean to make this comment on the SHA PR? I can do that there if you like.

Reply:
Well, yes, but we should always have `asm volatile`